### PR TITLE
kola: emit tap results

### PR DESCRIPTION
**Pull request by mischief**
initial go at tap.

we should really not filter tests and simply emit SKIP's for them so that the number of tests are consistent...

### CODE REVIEW

**Comment by mischief** [comment]
```
$ cat aws.tap 
TAP version 13
1..1
not ok 1 - docker.resources # [0] failed to run "docker run --rm --memory-swappiness=50 sleep sleep 0.2": output: "" status: "Process exited with: 2. Reason was:  ()" [1] failed to run "docker run --rm --memory-swappiness=50 sleep sleep 0.2": output: "" status: "Process exited with: 2. Reason was:  ()"
```

File: kola/harness.go [hunk 1]
```diff
@@ -78,6 +80,37 @@ type result struct {
...
+	}
+
+	var werr error
+
+	for i, r := range res {
+		t := r.test
+		err := r.result
+		if err != nil && err == register.Skip {
+			// TODO: print why it was skipped
+			_, werr = fmt.Fprintf(w, "not ok %d - %s # SKIP\n", i+1, t.Name)
```
Comment (marineam) [inline kola/harness.go]: shoudn't skip be 'ok'?

**Comment by mischief** [comment]
i misread `DIRECTIVES`.. fixed.

**Comment by marineam** [comment]
LGTM

**Pull request merged**